lib/robots/dor_repo/speech_to_text/stt_create.rb

```diff
@@ -27,8 +27,7 @@ def perform_work
         def send_sqs_message
           message_body = {
             id: job_id,
-            druid:,
-            media:
```

Comment:
so this got removed in the documentation, but afaict from the code and from testing locally yesterday evening, the `media` field is actually still required in the message body: https://github.com/sul-dlss/speech-to-text/blob/1fd0eb35b721349c75c75be5151dffc8564d4084/speech_to_text.py#L80-L85

i assume the removal from STT documentation was a mistake or a leftover from a discarded development path with the code.  so adding that field back was one of the little touchups i was going to make (to the speech-to-text README) in the small follow-on PR i mentioned this morning at standup.

Reply:
(or, maybe you and ed have discussed and this lines up with a coming change in the speech-to-text python worker?)

Reply:
Ok, I hadn't further discussed this was @edsu - was operating under the assumption that the whipser container would look for all files that matched the key for the message sent.  

BUT... we could also continue to send the media key here in the message body, and it could basically do this for the container (i.e. look at what is in S3 for that job_id and then send all of those files in the message body)

Reply:
yeah, i could see either approach, and don't think i have a well thought out preference.  seems like an easy adjustment either way... i might bias towards assuming for now that the current worker code is right... i just put up a small PR to that effect in speech-to-text: https://github.com/sul-dlss/speech-to-text/pull/22

unless @edsu has this change to just list bucket contents in the works already?

i think we discussed this at the end of last week, but i can't remember if we settled on an approach or which way we decided if so.

i have to grab lunch after this DLSS all staff that's happening right now, and it'll be 5 eastern time once that's over anyway... maybe we can confer and decide (or remind me of last week's decision 😅) in the morning?  seems like waiting till the morning shouldn't leave anyone seriously blocked this afternoon?